velox/exec/tests/TableEvolutionFuzzer.cpp

```diff
@@ -627,22 +705,46 @@ void TableEvolutionFuzzer::run() {
     applyRemainingFilters(
         generatedRemainingFilters,
         columnNameMapping,
-        pushownConfig,
+        pushdownConfig,
         subfieldFilteredFields);
   }
 
+  // Enable aggregation testing (50% chance)
+  std::optional<AggregationConfig> aggConfig;
+  bool shouldTestAggregation = folly::Random::oneIn(2, rng_);
```

Comment:
Let's make the probability a gflag and default to something like 0.2 (aggregation pushdown is relatively rare in prod so we don't need to test that so often as 0.5)